docker/linux-image.sh

```diff
@@ -156,7 +161,7 @@ main() {
         ;;
     x86_64)
         arch=amd64
-        kernel="${kversion}-amd64"
+        kernel="${kmajor}\.${kminor}\.${kpatch}-.*-amd64"
```

Comment:
I don't know why, but apt was unable to find the original `5.10.0-26-amd64` package. Maybe it disappeared or something? Anyway, this fixed it and unblocked me